### PR TITLE
bbackupp lastest (new cask)

### PR DESCRIPTION
**Pull request by Zach677**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

repoen https://github.com/Homebrew/homebrew-cask/pull/181323

### CODE REVIEW

File: Casks/b/bbackupp.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+cask "bbackupp" do
+  version "2.13.73"
+  sha256 "f48b40c5d97d2f196730e51617de72b7d3526afcf5b8fd0e5fc67ba8a4807577"
+
+  url "https://github.com/Lakr233/BBackupp/releases/download/#{version}/BBackupp.zip"
+  name "BBackupp"
+  desc "Automated iOS Backup Robot"
```
Comment (khipp) [inline Casks/b/bbackupp.rb:7]:
```suggestion
  desc "iOS device backup software"
```

**Review by khipp — COMMENTED** [review]
Thank you, @Zach677! I only have a minor suggestion regarding the cask's description.

**Comment by Zach677** [comment]
fine

File: Casks/b/bbackupp.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  depends_on macos: ">= :ventura"
+
+  app "BBackupp.app"
+
+  zap trash: [
+    "~/Documents/BBackupp",
+    "~/Library/HTTPStorages/wiki.qaq.BBackupp",
+    "~/Library/HTTPStorages/wiki.qaq.BBackupp.binarycookies",
+    "~/Library/Preferences/wiki.qaq.BBackupp.plist",
```
Comment (khipp) [inline Casks/b/bbackupp.rb:23]:
```suggestion
    "~/Library/Preferences/wiki.qaq.BBackupp.plist",
    "~/Library/Saved Application State/wiki.qaq.BBackupp.savedState",
```

**Review by khipp — APPROVED** [review]
Thank you, @Zach677!

**Comment by Zach677** [comment]
Thanks for all the help @khipp!

**Pull request merged**